frontend/src/components/login/login.js

```diff
@@ -16,13 +16,14 @@ function Login({toggleModalBox, setDisplayModal}) {
 
   async function postLogin() {
     const param = {
-      'username': username,
-      'password': password,
-      'email': "",
-
+    
```

Comment:
Could we finaly have things related to the api in the `frontend/src/api/djangoAPI.js` file?
I remember it cost be some trouble the other day to find things related to login/logout etc...